src/app/services/ninja-service.ts

```diff
@@ -69,4 +69,21 @@ export class NinjaService {
     }
     return predictions;
   }
+
+  getOperations(pathwayPrediction: PathwayPredictionResult): DeCaF.Operation[] {
+    if (pathwayPrediction.method === 'PathwayPredictor+OptGene') {
+      return pathwayPrediction.knockouts.map((gene) => Object.assign({
+        data: null,
+        id: gene,
+        operation: 'knockout',
+        type: 'gene',
+      }));
+    }
+    return pathwayPrediction.knockouts.map((reaction) => Object.assign({
```

Comment:
Maybe better to have an explicit `else if` here for the other method, and raise an error if the method is not recognized? In case for example new methods are added in the future and we forget to update this part of the code.

Reply:
That's a good point, thanks, I'll add this check